molecule/multiserver/INSTALL.rst

```diff
@@ -0,0 +1,22 @@
+*******
```

Comment:
I don't think we need this guide two times. Could you please remove them both and instead add a small section about testing into the readme

Reply:
yes, I'll do this, thanks for the comment :)